platform_versioned_docs/version-23.4.0/launch/launchpad.mdx

```diff
@@ -5,95 +5,96 @@ date: "21 Apr 2023"
 tags: [launchpad]
 ---
 
-Seqera Platform Launch makes it easy for users without Nextflow expertise to configure pipeline parameters and launch workflows. View, configure, and launch pipelines on the **Launch Pipeline** page, accessed from your workspace **Launchpad**.
+View, configure, and launch pipelines from your workspace **Launchpad**.
 
 ## Launchpad
 
-The **Launchpad** enables workspace users to launch pre-configured pipelines, add new pipelines, or perform a quick launch of unsaved pipelines. Use the **Sort by:** drop-down to sort pipelines, either by name or most-recently updated.
+The **Launchpad** enables workspace users to launch pre-configured pipelines, add new pipelines, or perform a quick launch of unsaved pipelines. Use the **Sort by:** dropdown to sort pipelines, either by name or most-recently updated.
 
 :::note
 A pipeline is a repository containing a Nextflow workflow, a compute environment, and pipeline parameters.
 :::
 
-The list layout is the default **Launchpad** view. Use the toggle to the right of the **Search** field to switch between the list and tile views. Both views display the compute environment of each pipeline for easy reference.
+The list layout is the default **Launchpad** view. Use the toggle next to the **Search** field to switch between the list and tile views. Both views display the compute environment of each pipeline for easy reference.
```

Comment:
@llewellyn-sl, this is one of the few areas where access to inline SVGs might be helpful.